assets/cubyz/biomes/trans/savannah_widow.zig.zon

```diff
@@ -0,0 +1,55 @@
+.{
+	.properties = .{
+		.hot,
+		.dry,
+	},
+	.minHeight = 22,
+	.maxHeight = 40,
+	.chance = 0
+	.roughness = 1,
+	.hills = 5,
+	.music = "cubyz:EasternThought",
+	.interpolation = .linear,
+	.ground_structure = .{
+		"2 to 4 cubyz:limestone",
+	},
+	.structures = .{
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:dirt",
+			.chance = 0.5,
+			.width = 2,
+			.variation = 4,
+			.depth = 5,
+			.smoothness = 0.3,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:dry_grass",
+			.chance = 0.5,
+			.width = 3,
+			.variation = 5,
+			.depth = 1,
+			.smoothness = 0.2,
+		},
+		.{
+			.id = "cubyz:simple_tree",
+			.leaves = "cubyz:baobab_leaves",
+			.log = "cubyz:baobab_log",
+			.top = "cubyz:baobab_top",
+			.chance = 0.05,
+			.type = .round,
+			.height = 1,
+			.height_variation = 7,
+			.leafRadius = 2,
+			.leafRadius_variation = 1,
+		},
+		.{
+			.id = "cubyz:simple_vegetation",
+			.block = "cubyz:cactus",
```

Comment:
Why does the savannah widow have cacti, but not the base biome? I think the cacti look kind of nice in the biome I think it deserves to be in the base biome as well.

Reply:
Argh, you reordered them in the files.